modules/utils/CMakeSources.cmake

```diff
@@ -0,0 +1,28 @@
+set(MODULE_PATH
+        ${CMAKE_CURRENT_SOURCE_DIR}/claws/utils)
+
+set(MODULE_PUBLIC_HEADERS
```

Comment:
We can now since CMake 3.12 safely use GLOB_RECURSE with Dependent Options, the main advantage is that we didn't need anymore to modify a CMakeSources.cmake

Reply:
I feel like that would be abusing globbing out of laziness, but I don't really mind if you insist

Reply:
`The file(GLOB) and file(GLOB_RECURSE) commands learned a new flag CONFIGURE_DEPENDS which enables expression of build system dependency on globbed directory’s contents.`

As you wish